internal/utils/moduleversionlabels.go

```diff
@@ -0,0 +1,40 @@
+package utils
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/kubernetes-sigs/kernel-module-management/internal/constants"
+)
+
+func GetModuleVersionLabelName(namespace, name string) string {
+	return fmt.Sprintf("%s_%s_%s", constants.ModuleVersionLabelPrefix, namespace, name)
+}
+
+func GetModuleLoaderVersionLabelName(namespace, name string) string {
+	return fmt.Sprintf("%s_%s_%s", constants.ModuleLoaderVersionLabelPrefix, namespace, name)
+}
+
+func GetDevicePluginVersionLabelName(namespace, name string) string {
+	return fmt.Sprintf("%s_%s_%s", constants.DevicePluginVersionLabelPrefix, namespace, name)
+}
+
+func GetNamespaceNameFromVersionLabel(label string) (string, string, error) {
+	parts := strings.Split(label, "_")
```

Comment:
I think the new [`strings.Cut`](https://pkg.go.dev/strings#Cut) would work great here.

Reply:
Indeed, I missed that.